### ISSUE TITLE
Outreach to new agencies for Data.gov

### ISSUE THREAD

**Issue by hkdctol** [GSA/data.gov#1688]
## User Story

As Data.gov PMO, we want to conduct outreach to additional federal agencies so that more agencies get datasets into Data.gov.

## Acceptance Criteria
[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]
- [x] GIVEN there are federal agencies that have not yet published datasets on Data.gov\
 AND federal agencies are required to publish datasets under the OPEN Government Data Act \
  WHEN we have an overview session prepared \
  THEN we will conduct an overview session for new agencies\
  AND conduct an overview that will be recorded and available.

## Background
We conducted an overview session in 2019 and individual attempts to reach agencies. There have been changes in agency CDO designations. Some agencies have been added to Data.gov but there remain federal agencies that have not yet made contact with Data.gov.

## Sketch
- [x] Prepare overview session
- [x] Coordinate with OMB
- [x] Schedule overview
- [ ] Conduct overview session and recording
- [ ] Make recording available.

**Comment by hkdctol** [comment]
Coordinating timing with CDO Council. Outreach to agencies that are on CDO list where we have non-working data.json links.

**Comment by mogul** [comment]
Presentation likely to happen in the next month, if CDO folks are on top of things.

**Comment by hkdctol** [comment]
Working on picking a date and then CDO Council will send invite

**Comment by mogul** [comment]
Now waiting on the CDO Council invite

**Comment by hkdctol** [comment]
I'm moving this to blocked pending some more information on FY21 resources.

**Comment by mogul** [comment]
Still waiting on FY21 clarity.

**Comment by hkdctol** [comment]
Reached out to CDO Council small agency committee, awaiting scheduling.

**Comment by mogul** [comment]
Now scheduled for February 11th, blocking until then.

**Comment by hkdctol** [comment]
Marking this as done as as overview happened with small agency working group under CDO. No recording because it was the CDO Council's meeting. Will create a separate issue later about creating a recording that can be posted.
